mod/node-core/pkg/components/api.go

```
@@ -85,7 +85,7 @@
 	],
 ) *backend.Backend[
 	AvailabilityStoreT, BeaconBlockT, BeaconBlockBodyT, BeaconBlockHeaderT,
-	BeaconStateT, BeaconStateMarshallableT, BlobSidecarsT, BeaconBlockStoreT,
+	BeaconStateT, BeaconStateMarshallableT, BlobSidecarT, BlobSidecarsT, BeaconBlockStoreT,
```

golangci-lint

the line is 88 characters long, which exceeds the maximum of 80 characters.

(lll)